pandas/src/generate_code.py

```diff
@@ -4,6 +4,11 @@
 # or we get a bootstrapping problem
 from StringIO import StringIO
 
+MAX_INT8 = 127
```

Comment:
use the dtype properties instead of hard coding these, e.g., https://github.com/pydata/pandas/blob/484f66814a32324248e3d03203ea991ed896fd7a/pandas/core/common.py#L52